src/main.py

```diff
@@ -86,7 +86,7 @@ def fn(soup):
     "Research article": "research-article",
     "Short report": "short-report",
     "Short Report": "short-report",
-    "Tools and Resources": "tools-resources",
```

Comment:
You could leave in this line if you want, since it is possible to have multiple phrases mapping to the same display channel.

Reply:
Thanks - Re-added in the latest commit.